packages/terra-tabs/src/common-tabs/_Tabs.jsx

```diff
@@ -66,6 +67,10 @@ const propTypes = {
   * Callback function triggered when tab is drag and dropped .
    */
   onTabOrderChange: PropTypes.func,
+  /**
+   * intl object programmatically imported through injectIntl from react-intl.
+   * */
```

Comment:
```suggestion
   */
```
shouldn't this be private prop ?